### DISCUSSION TITLE
Open in different editor

### DISCUSSION THREAD

**Discussion by lttr** [nuxt/devtools#371] [Q&A]
Can I configure what command is executed for opening files (e.g. using the locator tool to open a SFC)?

Specifically I want to open a file in a running neovim session.

I can already do it in the terminal by first opening neovim with `nvim --listen /tmp/nvimsocket` and later running `nvr my-file.ts`. `nvr` stands for neovim-remote, which is this script: https://github.com/mhinz/neovim-remote

**Comment by greabock** [comment]
Any update on this?

**Comment by arashsheyda** [comment]
have you tried this: https://github.com/nuxt/devtools/issues/576#issuecomment-2017681318
